xwiki-commons-core/xwiki-commons-xml/src/test/java/org/xwiki/xml/internal/html/DefaultHTMLCleanerTest.java

```diff
@@ -409,10 +409,12 @@ public void verifyEntitiesAreNotBroken()
     @Test
     public void verifyFormValuesAreNotTrimmed()
     {
+        assertHTML("<p><input type=\"hidden\" value=\"  fff\" /></p>", "<input type=\"hidden\" value=\"  fff\" />");
         assertHTML("<p><input type=\"hidden\" value=\"foo\" /></p>", "<input type=\"hidden\" value=\"foo\" />");
         assertHTML("<p><input type=\"hidden\" value=\"foo bar\" /></p>", "<input type=\"hidden\" value=\"foo bar\" />");
-        assertHTML("<p><input type=\"hidden\" value=\"  fff\" /></p>", "<input type=\"hidden\" value=\"  fff\" />");
-        assertHTML("<p><input class=\"  fff\" type=\"hidden\" /></p>", "<input type=\"hidden\" class=\"  fff\" />");
+        assertHTML("<p><input class=\"fff\" type=\"hidden\" /></p>", "<input type=\"hidden\" class=\"  fff\" />");
+        assertHTML("<p><input class=\"foo bar\" type=\"hidden\" value=\" foo bar  \" /></p>",
+            "<input type=\"hidden   \" value=\" foo bar  \" class=\" foo bar  \"/>");
```

Comment:
Do we have another test where we verify that attributes are trimmed by default (when they're not INPUT tag elements)?

Reply:
Actually in this test I'm checking the behaviour on different kind of attributes. But sure I can add one assert with a div if you like.

Reply:
Maybe I'm not seeing clear but I see only `<input>` tests.

Reply:
And the test says `verifyFormValuesAreNotTrimmed`.

Reply:
Yep input but you can see that I'm checking the class and type attributes too.

Reply:
Added an assert and changed the test name.